core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -184,6 +189,12 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
           error("whitelist must be specified when using new consumer in mirror maker.")
           System.exit(1)
         }
+
+        println("WARNING: The default partition assignment strategy of the new-consumer-based mirror maker will " +
+          "change from 'range' to 'roundrobin' in the future (so that a better load balancing can be achieved). " +
+          "One easy way to keep using your current default strategy ('range') is to add the following to the new " +
```

Comment:
So instead of advising users how to stick with `range` we advise how they can switch to `round-robin` in advance. I'll change the message. I'll also look into how we can be smart about when to print it, as you suggested. Thanks.